_includes/live-stream-embed.html

```diff
@@ -0,0 +1,34 @@
+<div class="stream-container">
+  <video id="live" class="video-js vjs-default-skin vjs-big-play-centered" data-setup='{"fluid": true}'
+   controls preload autoplay loop>
+    <p class="vjs-no-js">To view this video please enable JavaScript, and consider upgrading to a web browser that supports HTML5 video</p>
+  </video>
+
+  <div class="stream-selector" id="StreamSelector">
+    <div id="SelectStream">
+      <h2>Select Live Stream Source</h2>
```

Comment:
This could just be "Select Stream" or "Select Source" (we already have live stream above so maybe could be trimmed)